src/app/services/http.service.ts

```diff
@@ -65,6 +65,15 @@ export class HttpService {
     );
   }
 
+  getPaginatedHardworkHistoryData(page_size: number = 10, page_number: number = 0, vault?: string, minAmount: number = 0):
+  Observable<HardWorkDto[]> {
+    return this.http.get<HardWorkDto[]>
+    (`${this.apiEndpoint}/hardwork/pages?pageSize=${page_size}&page=${page_number}${vault ? `&vault=${vault}` : ''}&minAmount=${minAmount}`)
```

Comment:
add ordering (desc by default)

Reply:
I'll add it myself, no worries

Reply:
oh, I thought you said the endpoint was doing desc by default so I left it out. My bad!